### PR TITLE
Nc news ticket#10

### PR DESCRIPTION
**Pull request by reece-b4**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -187,34 +185,105 @@ describe("/api/articles/:article_id", () => {
...
+            title: expect.any(String),
+          article_id: expect.any(Number),
+        topic: expect.any(String),
+      created_at: expect.any(String),
+    votes: expect.any(Number),
+  comment_count: expect.any(Number)})
         })
       })
     })
+    test('200 - article exists but has no comments', ()=>{
```
Comment (daviidbartlett) [inline __tests__/app.test.js]: duplicate test?

File: models/articles.models.js [hunk 2]
```diff
@@ -35,3 +35,23 @@ exports.updateArticleById = (article_id, body) => {
...
+exports.selectCommentsByArticleId = (article_id) => {
+  return db.query(`SELECT * FROM comments
+  WHERE article_id = $1;`, [article_id])
+.then(({rows})=>{
+  return rows
+})
+};
+
+exports.selectArticles = () =>{
+  return db.query(`SELECT articles.*, COUNT(comments.body)::INT AS comment_count
```
Comment (daviidbartlett) [inline models/articles.models.js]: no body thanks!

**Review by daviidbartlett — COMMENTED** [review]
generally good! I think you tests have duplicated a bit though!

**Review by DCrawley94 — APPROVED** [review]
looks good!

**Pull request merged**